core-rust-bridge/src/main/java/com/radixdlt/rev2/ComponentAddress.java

```diff
@@ -125,6 +130,15 @@ public static void registerCodec(CodecMap codecMap) {
             0
           });
 
+  public static ComponentAddress ofAccountPublicKey(ECDSASecp256k1PublicKey publicKey) {
```

Comment:
I'd slightly rather this was done through a static call into rust, as I believe this mapping is going to change in future

Reply:
good point...and in fact it was already there :D `Address.virtualAccountAddress`